qa/automation/CareerPortal.Tests.Scenarios/1. Access to Career Portal/Grishniakova Send CV test/Feature 1.1. Send CV direct from careers.feature

```diff
@@ -0,0 +1,82 @@
+Feature 1.1. Send CV directly from the careers main page careers.quantori.com
+
+Scenario User sent CV 
+Given User filled in valid alphabetic name and surname
+And User filled in valid e-mail
+And User attached one file .RTF less than 10 mb
+And User put a tick in the checkbox
+When User pressed Send
+Then User's CV was sent and Thank you message displayed
+
+Scenario User did not fill in name
+Given User filled in valid surname with no name
+And User filled in valid e-mail
+And User attached one file .PDF less than 10 mb
+And User put a tick in the checkbox
+When User pressed Send
+Then the CV was not sent to Quantori and "Name" field higllighted with red with message "Invalid name format"
+
+Scenario User filled in name and surname in wrong format
+Given User filled invalid numeric name and surname
+And User filled in valid e-mail
+And User attached one file .PDF less than 10 mb
+And User put a tick in the checkbox
+When User pressed Send
+Then the CV was not sent to Quantori and "Name" field higllighted with red with message "Invalid name format"
+
+Scenario User filled in wrong email 
+Given User filled in valid alphabetic name and surname
+And User filled in invalid e-mail with no @
+And User attached one file .PDF less than 10 mb
+And User put a tick in the checkbox
+When User pressed Send
+Then the CV was not sent to Quantori and "E-mail" field higllighted with red with message "Invalid name format"
+
```

Comment:
The same about invalid data in the "Email" field - it could be better to combine these scenarios into Scenario Outline with examples and to separate its behavior from other fields and buttons since they don't effect given result (("Invalid name format" and a red line).

Reply:
Agreed!